go/vt/vtgate/evalengine/compiler_asm.go

```diff
@@ -3934,6 +3934,34 @@ func (asm *assembler) Fn_MAKETIME_i() {
 	}, "FN MAKETIME INT64(SP-3) INT64(SP-2) INT64(SP-1)")
 }
 
+func (asm *assembler) Fn_MAKETIME_D() {
```

Comment:
We're using a lowercase `d` as the suffix for other decimal related functions, so let's also keep using that here.

Reply:
we already have `Fn_MAKETIME_d` for decimal cases, `Fn_MAKETIME_D` is for the temporal cases, like the one in DATEADD `Fn_DATEADD_D`

Reply:
@beingnoble03 ah, sorry, you're right!